{
  "id": "CVE-2024-33863",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2024-05-14T16:17:22.170",
  "lastModified": "2024-07-03T01:59:02.140",
  "vulnStatus": "Awaiting Analysis",
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "An issue was discovered in linqi before 1.4.0.1 on Windows. There is /api/Cdn/GetFile local file inclusion."
    },
    {
      "lang": "es",
      "value": "Se descubri\u00f3 un problema en linqi anterior a 1.4.0.1 en Windows. Hay inclusi\u00f3n de archivos locales /api/Cdn/GetFile."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "NONE",
          "scope": "UNCHANGED",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "HIGH",
          "availabilityImpact": "HIGH",
          "baseScore": 9.8,
          "baseSeverity": "CRITICAL"
        },
        "exploitabilityScore": 3.9,
        "impactScore": 5.9
      }
    ]
  },
  "weaknesses": [
    {
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
      "type": "Secondary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-98"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://linqi.help/Updates/en#/SecurityUpdates",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://www.linqi.de/de-DE/blog.html",
      "source": "cve@mitre.org"
    }
  ]
}